packages/browserify/test/config.js

```diff
@@ -13,16 +13,19 @@ test('policy - default policy path is generated with autoconfig if path is not s
     opts: {
       writeAutoPolicy: true
     },
-    contextName: 'browserify'
+    contextName: 'browserify',
+    testType: 'falsy'
   })
   const { projectDir, policyDir } = await prepareScenarioOnDisk({ scenario, policyName: 'browserify' })
   const expectedPath = path.join(policyDir, 'policy.json')
 
-  t.falsy(fs.existsSync(expectedPath), 'Config file does not yet exist')
+  scenario.checkResult(t, fs.existsSync(expectedPath), scenario)
```

Comment:
dont use checkResult to assert an unrelated value (`fs.existsSync(expectedPath)`)
just keep the original assertion